package.json

```diff
@@ -6,10 +6,10 @@
     "dev": "parcel serve index.html",
     "lint-all-check": "npm-run-all -s --continue-on-error lint-tsc-check lint-eslint-check lint-prettier-check",
     "lint-all-fix": "npm-run-all -s lint-eslint-fix lint-prettier-fix",
-    "lint-eslint-check": "eslint --ignore-path .gitignore '**/*.{js,ts,tsx}'",
-    "lint-eslint-fix": "eslint --fix --ignore-path .gitignore '**/*.{js,ts,tsx}'",
-    "lint-prettier-check": "prettier --check --ignore-path .gitignore .",
-    "lint-prettier-fix": "prettier --write --ignore-path .gitignore .",
+    "lint-eslint-check": "eslint --ignore-path .gitignore 'src/**/*.{js,ts,tsx}'",
+    "lint-eslint-fix": "eslint --fix --ignore-path .gitignore 'src/**/*.{js,ts,tsx}'",
```

Comment:
Can you make sure eslint in `backend/functions` is run by the CI? I've had issues with where lint issues go unnoticed (maybe some people don't have their editor set up to show them), and this helps us see it on commit and in PRs.